src/nrnoc/init.cpp

```diff
@@ -1111,9 +1111,16 @@ double nrn_call_mech_func(Symbol* s, int narg, Prop* p, int type) {
 }
 
 void nrnunit_use_legacy() {
+    hoc_warning("nrnunit_use_legacy() is deprecated as only modern units are supported.",
+                "If you want to still use legacy unit you can modify the nrnunit.lib with a "
+                "previous version.");
```

Comment:
I would say let's remove this part: no one will understand modifying nrnunit.lib. So let's keep only first simple part.

Reply:
Maybe instead suggest to downgrade to an `8.x` version.